health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreen/AppConfigurationWrapper.js

```
@@ -2,8 +1,13 @@
+import React, { createContext, useContext, useReducer, useState } from "react";
 import AppFieldScreenWrapper from "./AppFieldScreenWrapper";
 import { Loader } from "@egovernments/digit-ui-components";
 import SidePanel from "./SidePanel";
 import { useTranslation } from "react-i18next";
 import DrawerFieldComposer from "./DrawerFieldComposer";
+import SidePanel from "./SidePanel";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'SidePanel'. Consider to delete it or rename it.

'SidePanel' is defined here:


(lint/suspicious/noRedeclare)